contrib/packaging/bootc.spec

```diff
@@ -95,6 +95,13 @@ cat vendor-config.toml >> .cargo/config.toml
 rm vendor-config.toml
 
 %build
+# https://issues.redhat.com/browse/RHEL-116881
+%if 0%{?rhel} >= 10
```

Comment:
Wasn't this happening on c9s too? I suspect it has to do with the rust version, not the host OS version.

It seems simpler to just unconditionally do it on s390x anyways.

Reply:
Less broken things = better!